packages/apps/composer-crx/tsconfig.json

```diff
@@ -5,70 +5,65 @@
     "declaration": false,
     "declarationMap": false,
     "emitDeclarationOnly": false,
-    "lib": [
-      "DOM",
-      "ES6"
-    ],
-    "outDir": "./dist/types",
-    "rootDir": "src"
+    "lib": ["DOM", "ES6"],
+
+    "rootDir": "src",
   },
-  "include": [
-    "src"
-  ],
+  "include": ["src"],
   "references": [
     {
-      "path": "../../../tools/vite-plugin-icons"
+      "path": "../../../tools/vite-plugin-icons",
```

Comment:
if this is acceptable, can you update toolbox not to undo it? it creates a lot of diff spam, especially when it gets merged